MahApps.Metro/Properties/AssemblyInfo.cs

```diff
@@ -18,5 +18,5 @@
 [assembly: AssemblyTitleAttribute("MahApps.Metro")]
 [assembly: AssemblyDescriptionAttribute("Toolkit for creating Metro styled WPF apps")]
 [assembly: AssemblyProductAttribute("MahApps.Metro 1.1.3-ALPHA")]
-
+[assembly: AssemblyCompany("MahApps Metro")]
```

Comment:
Works for me!MahApps Team sounds good.

/Andreas




On Tue, Jun 16, 2015 at 6:28 AM -0700, "Brendan Forster" <notifications@github.com> wrote:












In MahApps.Metro/Properties/AssemblyInfo.cs:
> @@ -18,5 +18,5 @@
>  [assembly: AssemblyTitleAttribute("MahApps.Metro")]
>  [assembly: AssemblyDescriptionAttribute("Toolkit for creating Metro styled WPF apps")]
>  [assembly: AssemblyProductAttribute("MahApps.Metro 1.1.3-ALPHA")]
> -
> +[assembly: AssemblyCompany("MahApps Metro")]



@punker76 I think this could be simplified to MahApps or even MahApps Team - let me know what your thoughts are on this.



—
Reply to this email directly or view it on GitHub.